priv/templates/phx.gen.live/core_components.ex

```diff
@@ -252,9 +252,15 @@ defmodule <%= @web_namespace %>.CoreComponents do
   attr :errors, :list
   attr :rest, :global, include: ~w(autocomplete checked disabled form max maxlength min minlength
                                    multiple pattern placeholder readonly required size step)
+
+  attr :hidden_input, :boolean,
+    default: true,
+    doc: "wether to show or not a hidden input for the unchecked checkbox input"
```

Comment:
Unchecked checkboxes aren't sent by default so it needs a hidden input with the value of "false" to update the boolean value. This allows having or not having the hidden input.
When working with some forms that don´t use checkboxes for boolean values, the hidden input is optional.